lib/http3/common.c

```diff
@@ -321,6 +321,8 @@ static void control_stream_handle_input(h2o_http3_conn_t *conn, struct st_h2o_ht
 static void discard_handle_input(h2o_http3_conn_t *conn, struct st_h2o_http3_ingress_unistream_t *stream, const uint8_t **src,
                                  const uint8_t *src_end, int is_eos)
 {
+    if (src == NULL)
```

Comment:
```
if (src != NULL)
    *src = src_end;
```

Please let me know if you prefer this shorthand. I chose the early return style simply just for consistency with the neighboring functions.

Reply:
Let's keep the code as-is.

We use early return is because our mental model is that receiving a reset is exceptional.